tutorials/coral-talk-on-cloud-run/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Coral Talk on Google Cloud Run
+description: How to deploy Coral Talk on Google Cloud Platform using managed services -  Cloud Run and Memorystore.
+author: vyolla
+tags: cloud-run, memorystore
+date_published: 2022-04-04
+---
+
+Bruno Patrocinio | Customer Engineer | Google
+
+
+<p style="background-color:#CAFACA;"><i>Contributed by Google employees.</i></p>
+
+This tutorial describes how to deploy open-source commeting platform, [Coral Talk](https://docs.coralproject.net/)  on Google Cloud Platform using managed services.
+
+The diagram below shows the general flow: 
+![architecture](images/architecture.png)
+
+The instructions are provided for a Linux development environment, such as [Cloud Shell](https://cloud.google.com/shell/).
+However, you can also run the application on Google Compute Engine, Kubernetes, a serverless environment, or outside of
+Google Cloud.
+
+This tutorial assumes that you know the basics of the following products and services:
+
+  - [Cloud Run](https://cloud.google.com/run/docs)
+  - [Container Registry](https://cloud.google.com/container-registry/docs)
+  - [Memorystore](https://cloud.google.com/memorystore/docs)
+  - [Compute Engine](https://cloud.google.com/compute/docs)
+  - [`gcloud`](https://cloud.google.com/sdk/docs)
+  - [Docker](https://docs.docker.com/engine/reference/commandline/run)
+
+## Objectives
+
+* Learn how to create and deploy services using `gcloud` commands.
+
+## Costs
+
+This tutorial uses billable components of Google Cloud, including the following:
+
+*   [Cloud Run](https://cloud.google.com/run)
+*   [Compute Engine](https://cloud.google.com/compute)
+*   [Memorystore](https://cloud.google.com/memorystore)
+
+Use the [Pricing Calculator](https://cloud.google.com/products/calculator) to generate a cost estimate based on your 
+projected usage. 
+
+This tutorial only generates a small amount of Cloud Run requests, which may fall within the free allotment.
+
+
+
+## Before you begin
+
+
+For this tutorial, you need a Google Cloud [project](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects).
+You can create a new one, or you can select a project that you have already created:
+
+1.  Select or create a Google Cloud project.
+
+    [GO TO THE MANAGE RESOURCES PAGE](https://console.cloud.google.com/cloud-resource-manager)
+
+2.  Enable billing for your project.
+
+    [ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
+
+3. Enable the Cloud Run and Container Registry APIs. For details, see [ENABLING APIs](https://cloud.google.com/apis/docs/getting-started#enabling_apis).
+   
+4. Add role *Container Registry Service Agent* in service account `[project-id]-compute@developer.gserviceaccount.com`
+
+
+## Detailed steps
+
+
+### Download images and upload to Google Container Registry
```

Comment:
Recommend using Artifact Registry instead unless there is some kind of limitation or requirement to use Container Registry.